experiments/scan_classes/sequences.py

```diff
@@ -229,8 +229,8 @@ def __init__(self, seqn_keys, sequence_slice_dict, sequence_interface, options):
 
 
         last_pulse = self.pulses[-1]
-        self.last_pulse_len = max([p['pulse_len'] + p['intra_pulse_start_time'] for p in
-                              self.pulses[-(last_pulse['combine_total']):]])
+        self.last_pulse_len = max([(p['pulse_len'] + p['intra_pulse_start_time']) for p in
+                              self.pulses[-(last_pulse['combine_total']):]])  # TODO does this work? pycharm saying max() will return a list
```

Comment:
maybe pycharm just got confused since there's a lot happening. Looks like it should work to us.